TROUBLESHOOTING.md

```diff
@@ -0,0 +1,192 @@
+# Troubleshooting Common Issues
+
+## Using this Document
```

Comment:
This might just be part of `npm doctor` -- see the thread https://github.com/npm/npm/pull/15756#discussion_r101497773

PRs welcome!